applications/nrf_desktop/src/modules/usb_state_pm.c

```diff
@@ -18,7 +18,7 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_USB_STATE_LOG_LEVEL);
 #include <caf/events/force_power_down_event.h>
 
 
-#define SUSPEND_DELAY K_MSEC(1000)
+#define SUSPEND_DELAY K_MSEC(3)
```

Comment:
Discussed offline. Consider removing delay if there is no reason to keep it.

Reply:
Change applied. The delay is removed and `power_down_event` is sent immediately.